src/transformers/models/auto/tokenization_auto.py

```diff
@@ -219,6 +219,13 @@
                     "CLIPTokenizerFast" if is_tokenizers_available() else None,
                 ),
             ),
+            # (
+            #     "flava",
+            #     (
+            #         "CLIPTokenizer",
+            #         "CLIPTokenizerFast" if is_tokenizers_available() else None,
+            #     ),
+            # ),
```

Comment:
Should be cleaned up or fixed. Looking at the processor file, it seems to be using the `BertTokenizer`?